docs/using-the-jdbc-driver/RdsProxy.md

```diff
@@ -0,0 +1,24 @@
+# RDS Proxy FAQ
+
+## What is Amazon RDS Proxy
+Amazon RDS Proxy is a fully managed, highly available database proxy for Amazon Relational Database Service (RDS). Similar to the AWS Advanced JDBC Driver, Amazon RDS Proxy also supports failover and is able to route requests to new database instances to reduce downtime.
+
+For more information, see the [[Amazon RDS Proxy FAQs](https://aws.amazon.com/rds/proxy/faqs/)].
+
+## What is the difference between Amazon RDS Proxy and AWS Advanced JDBC Driver
+* The RDS Proxy handles the infrastructure to perform connection pooling for the writer instance of the associated RDS or Aurora database. The AWS Advanced JDBC Driver does not implement connection pooling, therefore, user application needs to use a third party connection pool, such as HikariCP.
+* The AWS Advanced JDBC Wrapper only supports failover for Aurora database engines, whereas the Amazon RDS Proxy supports failover for all RDS database engines as well as the Aurora database engines.
+* The Amazon RDS Proxy detects transaction boundaries differently. The AWS Advanced JDBC Wrapper uses keywords such as COMMIT or ROLLBACK in the SQL statements, while Amazon RDS Proxy uses the network protocol to determine the transaction boundaries.
```

Comment:
That's is not clear and maybe misleading. I'm not sure how network protocol can help to determine transations.

Reply:
It is based on [this](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.howitworks.html#:~:text=RDS%20Proxy%20detects%20when%20a%20transaction%20ends%20through%20the%20network%20protocol%20used%20by%20the%20database%20client%20application.%20Transaction%20detection%20doesn%27t%20rely%20on%20keywords%20such%20as%20COMMIT%20or%20ROLLBACK%20appearing%20in%20the%20text%20of%20the%20SQL%20statement.), maybe instead of transaction boundaries I should say "when a transaction ends" instead?